### DISCUSSION TITLE
CVE-2022-35977 and other redis vuln not showing up in trivy scan

### DISCUSSION THREAD

**Discussion by prachimishra8** [aquasecurity/trivy#3788] [Q&A]
When I run trivy scan in https://github.com/redis/redis/ from my local trivy I do not get any redis related vul and the scan just says 
" Detecting pip vulnerabilities..." I am using trivy version **0.37.3.**
And when I read the release notes in redis it has many CVEs fixed https://github.com/redis/redis/releases?page=1 so trying to find a way to scan them and verify.

**Comment by knqyf263** [comment]
Trivy currently supports only `conan.lock` for C/C++. The Redis repository is not managed by conan.
https://aquasecurity.github.io/trivy/v0.38/docs/vulnerability/detection/language/

**Reply by prachimishra8** [reply]
Is there any ETA or plan for support of other package managers other than  conan.lock?

**Reply by knqyf263** [reply]
No, we don't have ETA at the moment.

**Reply by knqyf263** [reply]
I'll close this discussion. You can raise an issue if you have a specific request regarding other package managers.
